src/cmake/externalpackages.cmake

```diff
@@ -226,6 +226,10 @@ checked_find_package (fmt REQUIRED
 get_target_property(FMT_INCLUDE_DIR fmt::fmt-header-only INTERFACE_INCLUDE_DIRECTORIES)
 
 
+# Ultra HDR
+checked_find_package (libuhdr)
+
+
```

Comment:
This is fine here, but maybe for organizational clarity, I would recommend finding libuhdr immediately after the search for JPEG, since they are intimately related?